features/guests_added.feature

```diff
@@ -34,4 +65,26 @@ Feature: See if you are attending an event
       | firstname    | Test           |
       | lastname     | Guest          |
       | email        | test           |
-    Then I should see "Your registration failed, please make sure your information is correct."
\ No newline at end of file
+    Then I should see "Your registration failed, please make sure your information is correct."
+
+
+
+
+
+
+  Scenario: Testing Pie charts
+  Given the following guests exist:
+      | firstname      | lastname         | email        | occupation           | gender             | department          | event_id|
+      | John           | Doe              | jd@gmail.com | Undergrad            | Male               | Chemistry           | 1       |
+      | Sara           | Sen              | ss@gmail.com | Graduate             | Female             | Chemical Engineering| 1       |
+      | Steph          | Curry            | sc@gmail.com | Postdoc              | Male               | Stem non-chemistry  | 1       |
+      | Tara           | Gon              | tg@gmail.com | Faculty Staff        | Prefer not to say  | Other               | 1       |
+      | Alex           | Morgan           | am@gmail.com | Other                | Female             | Other               | 1       |
+      | Sasha          | Downer           | sd@gmail.com | Postdoc              | Other              | Stem non-chemistry  | 1       |
+      
+      
+  When I follow "event_1"
+  Then I should see "Loading..."
```

Comment:
This is testing that Chartkick is loading up an interactive graph.  Unfortunately Capybara cannot interact with it beyond this.

Reply:
works for me